stdr_robot/include/stdr_robot/motion/motion_controller_base.h

```diff
@@ -27,6 +27,8 @@
 #include <geometry_msgs/Twist.h>
 #include <geometry_msgs/Pose2D.h>
 
+#define STDR_PI 3.14159265359
```

Comment:
You should better use the already defined STL's `M_PI` unless there is something special with that pi.